server/db/model/User.js

```diff
@@ -307,6 +307,20 @@ schema.methods.sendPasswordResetEmail = function() {
     });
 };
 
+schema.methods.checkGroup = function(checkGroup) {
+    if (checkGroup === 'all') {
```

Comment:
The user doesn't necessarily need to have a group, but if they are in a group, any of them should work. Don't really know if we need that check, but maybe `any` is better